src/components/App.js

```diff
@@ -22,13 +24,21 @@ class App extends Component {
     const lat =  Number(user.data.attributes.lat)
     const lng = Number(user.data.attributes.long)
     const truckList = this.createLocationList(user)
-    this.setState({lat: lat, lng: lng, truckList:[...this.state.truckList, ...truckList ]})
+    this.setState({lat: lat, lng: lng, truckList:[...this.state.truckList, ...truckList ], trucks:[...this.state.trucks, ...user.data.attributes.trucks]})
   }
 
   loginUser = (userName) => {
     fetchUserName(userName)
-    .then(data => console.log(data))
-    .catch(error => console.log("error"))
+    .then(data => {
+      console.log('id', data.data.id)
+      const id = data.data.id
+      fetchTrucks(id)
+      .then(trucks => {
+        console.log('trucks', trucks.data)
+        this.setState({ trucks: trucks.data})
```

Comment:
you might need to pass the data from the first call into this fetch call to be able to use it

Reply:
other than the id that we are passing into it?

Reply:
nevermind on this, we need to have the lat and lng being returned from the api call